Update ACT Rules to use WAI-ARIA 1.2 (#1950)
act-rules/act-rules.github.io: _rules/aria-required-owned-element-bc4a75.md

#### Failed Example 1

> This element with the `list` role owns an element without any of its `listitem` [required owned elements][].

Reviewer: ```suggestion
This element with the `list` role owns an element which is not a `listitem` [required owned elements][].
```

#### Failed Example 6

This element with the `menu` role only owns an element with a `group` role. The `group` in turn owns an element with the `menuitem` role, and an element with the `group` role, in which each element has the `treeitem` role. ARIA `group` roles are allowed to own other elements with a `group` role, but those nested `group` nodes must still meet the requirements.

```html
<div role="menu">
	<div role="group">
		<span role="menuitem">Item 1</span>
		<div role="group">
			<span role="treeitem">Item 1</span>
			<span role="treeitem">Item 2</span>
		</div>
	</div>
</div>
```

#### Failed Example 7

> This element with the `list` role owns an element with the `listitem` role and an element with the `group` role, in which each element has the `listitem` role. The `group` role is no longer [required owned element][] for the `list` role.

Reviewer 2: ```suggestion
This element with the `list` role owns an element with the `listitem` role and an element with the `group` role, in which each element has the `listitem` role. The `group` role is no longer a [required owned element][] for the `list` role.
```
